pages/docs/tutorials/native/mpp-ios-android.md

```diff
@@ -0,0 +1,566 @@
+```
+
+### Tuning the Gradle Build Script
+We need to supply the right Framework depending on the selected target in the Xcode
+project. It also depends on the `Release` or `Debug` configuration. Lastly,
+we'd like to make Xcode compile the Framework for us before the build.
+We need to include the additional task to the end of the `SharedCode/build.gradle` file:
+
+<div class="sample" markdown="1" mode="groovy" theme="idea" data-highlight-only="1" auto-indent="false">
+
+```groovy
+task packForXCode(type: Sync) {
+    final File frameworkDir = new File(buildDir, "xcode-frameworks")
+    final String mode = System.getenv('CONFIGURATION')?.toUpperCase() ?: 'DEBUG'
+    final String target = System.getenv('SDK_NAME')?.startsWith("iphoneos") ? 'iOS' : 'iOSx64'
+
+    inputs.property "target", target
+    inputs.property "mode", mode
+    outputs.dir frameworkDir
+    dependsOn kotlin.targets."$target".compilations.main.linkTaskName("FRAMEWORK", mode)
+
+    from { kotlin.targets."$target".compilations.main.getBinary("FRAMEWORK", mode).parentFile }
+    into frameworkDir
+    
+    doLast {
+        new File(frameworkDir, 'gradlew').with {
+            text = "#!/bin/bash\nexport 'JAVA_HOME=${System.getProperty("java.home")}'\ncd '${rootProject.rootDir}'\n./gradlew \$@\n"
+            setExecutable(true)
+        }
+    }
+}
+
+tasks.build.dependsOn packForXCode
+```
+</div>
+
+Let's switch back to the Android Studio and execute the `build` target of the `SharedCode` project from
+the *Gradle* tool window. The task checks for environment variables set by the Xcode build and copies
+the right variant of the framework into the `SharedCode/build/xcode-frameworks` folder. We include the
+framework from the folder into the build:
+
+### Setting up Xcode Dependency for iOS Emulator
+
+Let's add the `SharedCode` framework to the Xcode project.
+For that let's click on the root node of the *project navigator* and select the *target* settings.
+Next, we click on the `+` in the *Embedded Binaries* section, click *Add Other...* button in the dialog
+to choose the framework from the disk. We point to the following folder: 
+```
+SharedCode/build/xcode-frameworks/SharedCode.framework
+```
+
+You shall see something similar to this: 
+![Xcode General Screen]({{ url_for('tutorial_img', filename='native/mpp-ios-android/xcode-general.png') }})
+
+We need to disable the *Bitcode* feature in the project too. Kotlin/Native produces the fully native
+binaries, not the LLVM bitcode, so we switch to the *Build Settings* tab and type `bitcode` into
+the search field. Select `No` for the *Enable Bitcode* option.
+
+![Xcode Build Settings]({{ url_for('tutorial_img', filename='native/mpp-ios-android/xcode-bitcode.png') }})
+
+Now we need to explain Xcode, where to look for frameworks. We need to add the full path to the 
+`SharedCode/build/xcode-frameworks` into the *Search Paths | Framework Search Paths* section.
```

Comment:
Btw, the resulting project seems to contain hard-coded paths after this.

Reply:
Adding `$(SRCROOT)/../../SharedCode/build/xcode-frameworks` instead should help.

Reply:
Thank you. Good catch